CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 
 * Respect `onPlatform` for groups.
 
+* Expose new matcher `doesNotComplete`
```

Comment:
Maybe "Add a `doesNotComplete` matcher that asserts that a Future never completes."? It's important to describe what new features do, not just that they exist.

Also, I'd move this above the previous changelog entry; I generally list features before bug fixes, because more people care about new features.